docs/user-guide/configuring/ignoring-code.md

```diff
@@ -0,0 +1,182 @@
+# Ignoring Code
+
+
+* [`ignorePatterns` in config files](#ignorepatterns-in-config-files)
+* [`.eslintignore`](#eslintignore)
+* [Using an Alternate File](#using-an-alternate-file)
+* [Using eslintIgnore in package.json](#using-eslintignore-in-packagejson)
+* [Ignored File Warnings](#ignored-file-warnings)
+* [Disabling Inline Comments](#disabling-inline-comments)
+
+## `ignorePatterns` in config files
+
+You can tell ESLint to ignore specific files and directories by `ignorePatterns` in your config files. Each value of `ignorePatterns` is the same pattern as each line of `.eslintignore` in the next section.
+
+```json
+{
+    "ignorePatterns": ["temp.js", "**/vendor/*.js"],
+    "rules": {
+        //...
+    }
+}
+```
+
+* The `ignorePatterns` property affects only the directory that the config file is placed in.
+* You cannot write `ignorePatterns` property under `overrides` property.
+* `.eslintignore` can override the `ignorePatterns` property of config files.
```

Comment:
I'm not sure if "override" is the correct word here. Do you mean if a `.eslintignore` file negates an ignore then it will override what's in the config file?

Reply:
Yes, I believe that's the intention here (e.g. using `.eslintignore` to unignore a specific file that's ignored by a glob pattern in `ignorePatterns`). Maybe something along the lines of the following would be clearer?

```suggestion
* Patterns defined in `.eslintignore` take precedence over the `ignorePatterns` property of config files.
```

Reply:
Actually, it's just combined. It's similar to:

```js
const patterns = [
    ...config.ignorePatterns,
    ...fs.readFileSync(".eslintignore", "utf8").split("\n")
]
```